CHANGELOG.md

```diff
@@ -189,6 +190,7 @@
 * [ENHANCEMENT] Update rollout-operator to `v0.7.0`. #5718
 * [ENHANCEMENT] Increase the default rollout speed for store-gateway when lazy loading is disabled. #5823
 * [BUGFIX] Fix compilation when index, chunks or metadata caches are disabled. #5710
+* [ENHANCEMENT] Autoscaling: treat OOMing containers as though they are using their full memory request. #5739
```

Comment:
Personally I consider this a bugfix. (But if it's not it needs moving with the other enhancements).

We can also add an enhancement here for scaling the ruler-qurier by memory.